.github/scripts/get_system_info.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+#  Copyright (C) 2023 Intel Corporation
+#  Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See LICENSE.TXT
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# get_system_info.sh - Script for printing system info
+
+function system_info {
+	echo "********** system_info **********"
+	cat /etc/os-release | grep -oP "PRETTY_NAME=\K.*"
+	echo "**********SYCL-LS**********"
+	source /opt/intel/oneapi/setvars.sh
+	sycl-ls
+	echo "*************** installed-packages ***************"
+	# Instructions below will return some minor errors, as they are dependent on the Linux distribution.
+	zypper se --installed-only 2>/dev/null || true
+	apt list --installed 2>/dev/null || true
+	yum list installed 2>/dev/null || true
+	echo "**********VGA**********"
+	lspci | grep VGA
+	echo "**********CUDA Version**********"
+	if command -v nvidia-smi &> /dev/null; then
+		nvidia-smi
+	else
+		echo "CUDA not installed"
+	fi
+	echo "**********L0 Version**********"
+	dpkg -l | grep level-zero
+	echo "**********ROCm Version**********"
+	if command -v rocminfo &> /dev/null; then
+		rocminfo
+	else
+		echo "ROCm not installed"
+	fi
+	echo "**********/proc/cmdline**********"
+	cat /proc/cmdline
+	echo "**********/proc/modules**********"
+	cat /proc/modules
+	echo "**********/proc/cpuinfo**********"
```

Comment:
instead of this we could actually use `lscpu`, it has much simpler output

Reply:
done